src/fd_monitor.rs

```diff
@@ -45,30 +45,30 @@ impl FdEventSignaller {
     /// The default constructor will abort on failure (fd exhaustion).
     /// This should only be used during startup.
     pub fn new() -> Self {
-        cfg_if!(
+        cfg_if! {
```

Comment:
ah I didn't know that cfg_if supports expressions.
But it doesn't help us much because `let x = cfg_if ...` doesn't work.